Example/src/ScrollableViewExample.js

```diff
@@ -128,10 +128,22 @@ function Box({ color }) {
 
 export default function Example() {
   return (
-    <ScrollableView>
-      {colors.map((color) => (
-        <Box color={color} key={color} />
-      ))}
-    </ScrollableView>
+    <View style={styles.wrapper}>
+      <ScrollableView>
+        {colors.map((color) => (
+          <Box color={color} key={color} />
+        ))}
+      </ScrollableView>
+    </View>
   );
 }
+
+const styles = StyleSheet.create({
+  wrapper: {
+    height:
+      Platform.OS === 'web'
+        ? windowDimensions.height - Header.HEIGHT
+        : undefined,
+    overflow: Platform.OS === 'web' ? 'hidden' : undefined,
```

Comment:
Setting `undefined` here uses a default value or just overwrite default with `undefined`?

Reply:
it assigns udefined

Reply:
So we actually want an undefined here?

Reply:
it's ok